ajustes paper

franciscomeneses opened the pull request:
Quedo pendiente (Lo vere hoy):

- hacer titulos para tablas html/pdf
- centrar ajustes en tabla de cargas factoriales

franciscomeneses committed:
ajustes paper

Quedo pendiente (Lo vere hoy):

- hacer titulos para tablas html/pdf
- centrar ajustes en tabla de cargas factoriales

juancarloscastillo reviewed (changes requested):
agregar pendientes

franciscomeneses committed:
Ajustes

Incluye:
- Nueva bibliografia
- nuevo parrafo al paper
- Explicacion invarianza entre ordenes.
- Grafico descriptivo de la escala
- Corrplot
- html con rev.biblio (provisoria)
-

Falta
- centrar fit (creo que no se puede)
- separar descriptivos de la tabla de correlaciones.
- titulos
- llamar imagenes guardadas (no he podido guardarlas)

franciscomeneses committed:
paper

paper

juancarloscastillo commented:
@franciscomeneses , voy a esperar mejor los gráficos likert para hacer el merge del pull request.

franciscomeneses committed:
Plotlikert

El grafico se genera en analisis, es enviado a output/images y llamado desde paper. subo ademas el .css que comente.

franciscomeneses committed:
Rederizado

 termino pendientes sobre las tablas.

aun existen problemas de compativilidad html/pdf, funcionan pero hay que mejorar los estilos. Tengo algunas ideas de como solucionar esto.

subo todos los documentos por que no se que es necesario o no para el renderizado.

franciscomeneses committed:
Render final

Se mejoran las tablas, los titulos y l pdf.
ademas se agrega la opcion pdf dentro de las pestañas del _site.yml

franciscomeneses committed:
Todo listo

- tablas y titulos en ambos formatos.
- graficos llamados como imagenes

franciscomeneses committed:
Renderizado

franciscomeneses committed:
Todo listo

Ya estan listas las tablas con los titulos y los demas detalles de formato

juancarloscastillo committed:
Merge branch 'master' into paper_cor
